modules/core/src/main/lilaism/LilaLibraryExtensions.scala

```diff
@@ -91,6 +91,16 @@ trait LilaLibraryExtensions extends CoreExports:
     def parallel[B](f: A => Fu[B])(using Executor, BuildFrom[M[A], B, M[B]]): Fu[M[B]] =
       Future.traverse(list)(f)
 
+    def parallelVoid[B](f: A => Fu[B])(using Executor): Fu[Unit] =
+      list.iterator
+        .foldLeft(fuccess(()))((fr, a) => fr.zipWith(f(a))((_, _) => ()))
+
+  extension [A, M[A] <: IterableOnce[A]](list: M[Fu[A]])
+
+    def parallelVoid(using Executor): Fu[Unit] =
+      list.iterator
+        .foldLeft(fuccess(()))((fr, fa) => fr.zipWith(fa)((_, _) => ()))
+
   // these futures have already started running in parallel.
```

Comment:
should We add
```scala
    def parallel(using Executor, BuildFrom[M[Future[A]], A, M[A]]): Fu[M[A]] =
      Future.sequence(list)
```
and remove these `parallel` functions?

Reply:
yes. We could remove most of them then.

Reply:
ok, next commit is comming